proto/config.proto

```diff
@@ -38,18 +42,18 @@ message LinuxConfig {
 // container is created for.
 message Platform {
 	// OS is the operating system.
-	optional string os = 1;
+	string os = 1;
```

Comment:
since this is pretty static, and validation would be nice, it might be worth considering an enum:

```
enum OS {
  OS_NONE = 0; // I always have a zero value to denote the field not being set, proto3 requires a zero value.
  OS_DARWIN = 1;
  ...
}

enum Arch {
  ARCH_NONE = 0; // also note proto uses c++ enum name rules, so my general practice is to preface any enum names with the type name to avoid future issues.
 ...
}
```